Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```
@@ -160,15 +160,14 @@
     - id: RCDAmmo
     - id: AirGrenade
     - id: InsulatedGlovesAtmos #sunrise-edit
-
 - type: entityTable
   id: FillAtmosphericsHardsuit
   table: !type:AllSelector
     children:
     - id: ClothingOuterHardsuitAtmos
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)